content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Consequences of abortion bill on privacy in U.S. and E.U."
+date: 2022-11-18
+drafts: false
+---
+
+#### Recalling the abortion topic and situation
+
+The U.S. Supreme Court overturned Roe v. Wade on June 24, removing the right to abortion at the federal level and leaving it up to each U.S. state to decide whether to allow or criminalize it. Since this, 17 states forbid or limit abortion. Some members of the anti-abortion movement in the United States want to “restrict people from crossing state lines for abortions”, reports The Washington Post[^1]. This is of course a problem for the US democracy that has different implications that go far beyond that.
+
+First, but this is not the goal of this post, this story is also a warning for other countries as it reveals that even if, most of the time, democracies go forward regarding people’s rights, a return to the past is always possible, even for important humans rights that we may too easily take for granted. A first incident happenned on June 30th when an Australian girl from Brisbane, during a stopover in Los Angeles, was asked “Have you recently had an abortion?”. She said "no" but this answer seemed suspicious and she was placed in a detention room, interrogated twice, frisked, fingerprinted and photographed. Finally she learned that she would not be allowed to enter US territory, and that she would be deported on the next flight back to Brisbane...[^2]
```

Comment:
Going backwards rather than return to the past? I'm not sure though.

Reply:
```suggestion
First, but this is not the goal of this post, this story is also a warning for other countries as it reveals that even if, most of the time, democracies go forward regarding people’s rights, a return to the past is always possible, even for important humans rights that we may too easily take for granted. A first incident happenned on June 30th when an Australian girl from Brisbane, during a stopover in Los Angeles, was asked “Have you recently had an abortion?”. She said "no" but this answer seemed suspicious and she was placed in a detention room, interrogated twice, frisked, fingerprinted and photographed. Finally she learned that she would not be allowed to enter US territory, and that she would be deported on the next flight back to Brisbane…[^2]
```

Reply:
I put "return backwards" and split the pear :). Punctuation corrected too.